services/queries.sql

```diff
@@ -0,0 +1,5 @@
+gender_query = 
```

Comment:
Let's make this a configuration file of the format (yaml)
```
- tableId: gender
  queries:
    - query: "select concept_id,concept_name from concepts where vocabulary_id='gender' and concept_id=$concept_id"
       columns: 
             - concept_id
             - concept_name
    - query: "select concept_id,concept_name from concepts where vocabulary_id='gender' and concept_name=$concept_name"
       columns: 
             - concept_id
             - concept_name
```
The way it operates is this:
 - Match the id of the request with the tableId
 - Run the given queries one by one until a result is found
 - For each query, fill $varName with the value of the request.keys[varName]
 - If the query returns results, return result with key-value pairs constructed from the columns, with columns[i] for key, queryResult.Column[i] for value, etc.